README.md

```diff
@@ -557,6 +558,262 @@ node('windows') {
 ```
 </details>
 
+### Configuration as Code
+
+The [Jenkins Configuration as Code Plugin](https://plugins.jenkins.io/configuration-as-code) (a.k.a. JCasC) allows to configure Jenkins based on human-readable declarative configuration files. This plugin supports the external configuration of ECU-TEST tool installations and of TEST-GUIDE server instances in the Jenkins global configuration.
+
+Existing configurations of both types can be exported to according YAML file using the [JCasC export view](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/configExport.md).
+
+<details>
+    <summary>ECU-TEST installation</summary>
+
+```yml
+tool:
+  ecu-test:
+    installations:
+    - name: "ECU-TEST 8.0"
+      home: "C:\\Program Files\\ECU-TEST 8.0"
+    - name: "ECU-TEST 8.1"
+      home: "C:\\Program Files\\ECU-TEST 8.1"
+      properties:
+      - ecuTestProperty:
+          progId: "ECU-TEST.Application.8.1"
+          registerComServer: true
+          timeout: 60
+```
+</details>
+
+<details>
+    <summary>TEST-GUIDE server</summary>
+
+```yml
+unclassified:
+  testGuide:
+    installations:
+    name: "TEST-GUIDE"
```

Comment:
`name` & `toolName` are  still under installations key not as in the exported configuration unde config. Bug in exporter or bug in yaml file?:
```
   installations:
        - config:
              name: "TEST-GUIDE"
              toolName: "ECU-TEST 8.0"
              settings:
                  - atxBooleanSetting:
```

Reply:
Fixed